assignment-client/src/avatars/AvatarMixerSlave.cpp

```diff
@@ -275,18 +284,9 @@ void AvatarMixerSlave::broadcastAvatarDataToAgent(const SharedNodePointer& node)
     // setup a PacketList for the avatarPackets
     auto avatarPacketList = NLPacketList::create(PacketType::BulkAvatarData);
 
-    // Define the minimum bubble size
-    const glm::vec3 minBubbleSize = avatar.getSensorToWorldScale() * glm::vec3(0.3f, 1.3f, 0.3f);
-    // Define the scale of the box for the current node
-    glm::vec3 nodeBoxScale = (avatar.getClientGlobalPosition() - avatar.getGlobalBoundingBoxCorner()) * 2.0f * avatar.getSensorToWorldScale();
```

Comment:
This calculation for nodeBoxScale was just WRONG.  In particular: applying the `sensorToWorldScale` is incorrect.  The only reason it was never noticed is because for most cases the `sensorToWorldScale` is 1.0.